source/speech/__init__.py

```diff
@@ -887,21 +959,36 @@ def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=cont
 				presCat=field.getPresentationCategory(newControlFieldStack[0:count],formatConfig,reason)
 				if not presCat or presCat is field.PRESCAT_LAYOUT:

```

Comment:
Just a suggestion:

Initially, I thought that the presentation category code might cause the section reporting, but it is untouched. Yet, I'd like to point your attention at it. `textInfos.ControlField.PRESCAT_LAYOUT` is None, which to me is pretty confusing. may be it should be changed to `"layout"` so we can use proper type checking to avoid issues now or in the future.

Reply:
This is a good point. I would be with you on this idea, I'd be interested to know what @jcsteh thinks though? 

Jamie:
In short the value for `textInfos.ControlField.PRESCAT_LAYOUT` is `None`, how risky would it be to give this an actual default value like "layout"? Is there a reliance on not being able to retrieve an attribute or dictionary value and therefore `None` matches the intended default?

Happy for the answer to be "Not sure, please investigate" but if you know off-hand it might save us time.